.travis.yml

```diff
@@ -1,10 +1,14 @@
 sudo: required
+env:
+- PUBLIC_REVERSE_DEPENDENCIES=cli
 services: docker
 script: docker build . -t cs50/baseimage
 before_deploy: docker login -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD"
 deploy:
-  provider: script
+- provider: script
   script: docker push cs50/baseimage
+- provider: script
+  script: 'for repo in $PUBLIC_REVERSE_DEPENDENCIES; do curl -s -X POST -H "Content-Type: application/json" -H "Accept: application/json" -H "Travis-API-Version: 3" -H "Authorization: token $TRAVIS_PUBLIC_TOKEN" -d "{ \"request\": { \"branch\": \"ubuntu\" } }" "https://api.travis-ci.org/repo/cs50%2F$repo/requests"; done'
```

Comment:
Should we submit a feature request to make this a .travis.yml feature (to simplify)?

Reply:
They said it was on their road map https://github.com/travis-ci/travis-ci/issues/249#issuecomment-84052187. But sure, we could bring it up. I think it would be better if we factor out into bash scripts per https://github.com/cs50/cli/pull/24.

Reply:
Where would the script live?

Reply:
The deployment script itself could live in a repo or a gist. I suppose we could set the environment variables in the repo settings in Travis CI, but if that doesn't work for some reason, we could store them in a script in a dot folder or something.

Reply:
Eh, it seems okay just to leave it as a one-liner for now. Only need this in 3 child images, so feels okay to just keep local rather than add an external dependency?

Reply:
No problem.